src/customizations/volto/components/manage/Sidebar/ObjectBrowserBody.jsx

```diff
@@ -106,7 +109,7 @@ class ObjectBrowserBody extends Component {
     super(props);
     this.state = {
       currentFolder:
-        this.props.mode === 'multiple' ? '/' : this.props.contextURL || '/',
+        this.props.mode === 'multiple' ? this.props.contextURL || '/' : '/',
```

Comment:
Così facendo non è più contestuale la ricerca quando è singola. A questo punto farei `this.props.contextURL || '/'` per tutti e siamo a posto. O mi perdo qualcosa per strada?